src/main/java/de/tum/in/www1/artemis/web/rest/metis/conversation/ConversationResource.java

```diff
@@ -167,4 +168,17 @@ private void checkEntityIdMatchesPathIds(Conversation conversation, Optional<Lon
             }
         });
     }
+
+    /**
+     * GET /conversations/for-notifications
```

Comment:
This is a bad documentation. It does not tell why this REST call is even necessary. I don't see a valid reason.
It also seems to be problematic when there are many notifications / conversations. Does this scale when a tutor has a lot of communication going on in a course?

Reply:
I tested this with a tutor which has 3 courses with 50 conversations each and the response time was 30.42ms and a student with 0 conversations and the response time was 31ms. 

 I think there will be a small difference when conversations have 1000 students each but still this call will be only once when user opens the page or when on-reload, so IMO it should be fine.
 
 
Instructor with 150 conversations: 
![Screenshot 2023-04-16 at 02 45 15](https://user-images.githubusercontent.com/26312757/232264049-0d12ee23-ed89-46c8-b719-2889a367e19d.jpg)

Student with 0 conversation:
![Screenshot 2023-04-16 at 02 46 19](https://user-images.githubusercontent.com/26312757/232264066-4b7cc29a-516e-4d34-aaa4-f1f3dd023d2c.jpg)